index.html

```diff
@@ -0,0 +1,1591 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="main.css" />
+    <title>TNW</title>
+    <script
+      src="https://kit.fontawesome.com/63c1245c39.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!-- Navigator -->
+    <nav class="nav nav-m-b">
+      <!-- Logo -->
+      <div class="nav__logo-box">
+        <a href="#" class="nav__logo-link">
+          <img
+            src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+            alt="TNW Logo"
+            class="nav__logo"
+          />
+        </a>
+      </div>
+
+      <ul class="nav__list-click">
+        <li class="nav__click-item">
+          <a href="#" class="nav__click">Latest</a>
+        </li>
+        <li class="nav__click-item">
+          <a href="#" class="nav__click">Hard Fork</a>
+        </li>
+        <li class="nav__click-item">
+          <a href="#" class="nav__click">Plugged</a>
+        </li>
+        <li class="nav__click-item">
+          <a href="#" class="nav__click">Readme</a>
+        </li>
+        <li class="nav__click-item">
+          <a href="#" class="nav__click">Growth Quarters</a>
+        </li>
+      </ul>
+
+      <!-- Navigator List Options-->
+      <div class="nav__list-box">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--search">
+              <i class="fas fa-search"></i>
+            </a>
+          </li>
+          <!-- Hamborger Icon :D -->
+          <li class="nav__item">
+            <a class="nav__icon hamborger">
+              <span class="hamborger__ele hamborger__ele--top"></span>
+              <span class="hamborger__ele hamborger__ele--center"></span>
+              <span class="hamborger__ele hamborger__ele--bottom"></span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- Main Conetent -->
+    <main>
+      <div class="slider__section">
+        <!-- Start The slider Block -->
+        <article class="slider">
```

Comment:
Please add the TNW logo.

| Source Page	| Clone Page |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/57726348/87091914-2280d300-c258-11ea-80c0-c1bb0040474d.png)| ![image](https://user-images.githubusercontent.com/57726348/87091975-3b898400-c258-11ea-9ade-87baca3a6853.png)|